prooph/service-bus-symfony-bundle/0.6/config/packages/prooph_service_bus.yaml

```diff
@@ -0,0 +1,12 @@
+prooph_service_bus:
+    command_buses:
+        app_command_bus: ~
+    event_buses:
+        app_event_bus: ~
+
+services:
+    _defaults:
+        autowire: true
+
+    Prooph\Bundle\ServiceBus\CommandBus: '@prooph_service_bus.app_command_bus'
```

Comment:
Maybe alias id should be `Prooph\ServiceBus\CommandBus` to be consistent with the following alias? Any thought @UFOMelkor?

Reply:
It will be used in user's controller:
```php

use Prooph\Bundle\ServiceBus\CommandBus;
// or 
use Prooph\ServiceBus\CommandBus;

function __invoke(CommandBus $bus)
{
   $bus->dispatch(...);
}
```

Reply:
`Prooph\Bundle\ServiceBus\CommandBus` is an implementation detail of the bundle and should not be used directly. We should tag it as internal.

-> Use `Prooph\ServiceBus\CommandBus`